nixos/modules/config/shells-environment.nix

```diff
@@ -169,7 +169,7 @@ in
          ${cfg.extraInit}
 
          # The setuid wrappers override other bin directories.
```

Comment:
not 100% sure but maybe we should update comments saying "setuid" to "setuid/setcap" ? don't do if you don't think it's a good idea.